src/coreclr/jit/compiler.h

```diff
@@ -2230,6 +2230,60 @@ typedef JitHashTable<GenTree*, JitPtrKeyFuncs<GenTree>, TestLabelAndNum> NodeToT
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 #endif // DEBUG
 
+//-------------------------------------------------------------------------
+// LoopFlags: flags for the loop table.
+//
+enum LoopFlags : unsigned short
```

Comment:
If you're already defining operators for the enum type below, it would be worthwhile to change this to an `enum class` instead of an unscoped `enum`. Though it's also ok if that's left for a later PR to avoid renaming concerns.

Reply:
Doesn't that require changing all uses from (e.g.) `LPFLG_DO_WHILE` to `LoopFlags::LPFLG_DO_WHILE`?

Reply:
Yes. That's why I mentioned this might be good for a follow up PR, in which you could make it an enum class and remove the `LPFLG_` prefix (since you'll need to use the `LoopFlags::` scope).

Since you mentioned you wanted to keep this change minimal, its probably a good idea to leave this for a followup. I just wanted to raise it to make sure it was considered.